test/functional/test_runner.py

```diff
@@ -153,6 +153,7 @@
     # These are python files that live in the functional tests directory, but are not test scripts.
     "combine_logs.py",
     "create_cache.py",
+    "example_test.py",
```

Comment:
Yes, good idea. `example_test` should be run in the extended test run to test the test.